cmd/reposeed/main.go

```diff
@@ -16,12 +16,14 @@ package main
 import (
 	"flag"
 	"fmt"
-	"text/template"
 	"io/ioutil"
 	"log"
 	"os"
 	"path/filepath"
+	"strings"
+	"text/template"
 
+	"github.com/erbesharat/reposeed/templates"
```

Comment:
Your own fork is used currently. Intentional?

Reply:
Nope, it was a mistake 😄